frontend/app/services/EventbriteService.scala

```diff
@@ -43,7 +43,7 @@ abstract class EventbriteService(implicit val ec: ExecutionContext, system: Acto
   def eventsTaskFor(status: String, initialDelay: FiniteDuration, refreshTime: FiniteDuration): ScheduledTask[Seq[RichEvent]] =
     ScheduledTask[Seq[RichEvent]](s"Eventbrite $status events", Nil, initialDelay, refreshTime) {
       for {
-        events <- getAll[EBEvent]("users/me/owned_events/", List(
+        events <- getAll[EBEvent]("organizations/104062388105/events/", List(
```

Comment:
I think we have more than one organisation - there's one for masterclasses and one for live events. I.e. the difference between:
https://membership.theguardian.com/events AND
https://membership.theguardian.com/masterclasses

Reply:
I believe I've got access to access these if you don't have one or the other of them.

Reply:
oh yes I can see that, I just looked at the main api key, good spot 🏅